def.bzl

```diff
@@ -8,128 +8,136 @@ project = struct(
 
     # External binaries; see external_binary() invocation in WORKSPACE.
     external_binaries = {
-        "bosh_cli": struct(
-            sha256 = {
+        "bosh_cli": {
```

Comment:
I guess this is still related to #545 . That's more a post-merge comment: since we (were) here, couldn't we have a yaml file that `bazel` is reading instead? Is Bazel implying `struct` in this case?

Reply:
Done with https://github.com/cloudfoundry-incubator/kubecf/pull/712. :)